src/components/body/index.js

```diff
@@ -0,0 +1,18 @@
+export function Body() {
+    return (
+        <div className='body'>
+          <div className='body-left'>
+          </div>
+          <div className='body-right'>
+            <div className="body-header">
+            I SEE SERVICE IN YOUR FUTURE
```

Comment:
Should never have all uppercase on the actual text. You use `css` to do that for us.
Instead of using `div`, you should use `h1`, since it's a heading.
```css
.body-header {
 text-transform: uppercase;
}
```